inference.py

```diff
@@ -1,79 +1,95 @@
 from time import sleep
 from fastai.vision.all import *
-from ue5env import UE5EnvWrapper
-
+from ue5osc import Communicator
 import os
 import pathlib
+from math import radians
+from argparse import ArgumentParser, Namespace
+import time
+
+# Need it otherwise we get an Error PoxisPath
 temp = pathlib.PosixPath
```

Comment:
There must be a better (less odd) way to do this. What is the error?

Reply:
raise NotImplementedError("cannot instantiate %r on your system"
NotImplementedError: cannot instantiate 'PosixPath' on your system